packages/tests/shadydom/active-element.html

```diff
@@ -496,6 +497,7 @@
     r_1_1.focus();
     try {
       ShadyDOM.wrapIfNeeded(ShadyDOM.wrapIfNeeded(r_1).shadowRoot).activeElement = "abc";
+    // eslint-disable-next-line no-empty
     } catch(e) {}
```

Comment:
```suggestion
    }
```

(also depends on polyserve)